src/riak_kv_select.erl

```diff
@@ -117,19 +134,36 @@ downgrade_select([v2,v1=To|Tail], Select1) ->
         cover_context = Select1#riak_select_v2.cover_context,
         local_key     = Select1#riak_select_v2.local_key
     },
+    downgrade_select([To|Tail], Select2);
+downgrade_select([v3,v2=To|Tail], Select1) ->
+    Select2 = #riak_select_v2{
+        'SELECT'      = Select1#riak_select_v3.'SELECT',
```

Comment:
Aaand it did cause trouble, was bound to. When a 1.4 node receives an assignment to spin up an index_fsm, it eventually trips on bad `?SQL_SELECT` field index (around here https://github.com/basho/riak_kv/blob/riak_ts-develop/src/riak_kv_qry_coverage_plan.erl#L80), leading to a cryptic crash with these.. words:
```
{function_clause,
 [{riak_ql_ddl, make_key,
   [{key_v1, [{param_v1, [<<"a">>]}, {hash_fn_v1, riak_ql_quanta, quantum, [{param_v1, [<<"b">>]}, 10, s], timestamp}]},
    true,
    [{<<"a">>, 1}, {<<"b">>, 1000}]],
   [{file, "src/riak_ql_ddl.erl"}, {line, 168}]},
  {riak_kv_qry_coverage_plan, make_key, 3, [{file, "src/riak_kv_qry_coverage_plan.erl"}, {line, 129}]},
  {riak_kv_qry_coverage_plan, do_plan, 7, [{file, "src/riak_kv_qry_coverage_plan.erl"}, {line, 83}]},
  {riak_kv_index_fsm, create_plan, 8, [{file, "src/riak_kv_index_fsm.erl"}, {line, 161}]},
  {riak_core_coverage_fsm, initialize, 2, [{file, "src/riak_core_coverage_fsm.erl"}, {line, 272}]},
  {gen_fsm, handle_msg, 7, [{file, "gen_fsm.erl"}, {line, 505}]}, {proc_lib, init_p_do_apply, 3, [{file, "proc_lib.erl"}, {line, 239}]}]}
```
A fix is to append, rather than insert, new fields to `#riak_select_v3` (a7eb41a),